src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -34,6 +34,17 @@ default boolean isSameStateAs(ReadOnlyPerson other) {
     }
 
     /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    default String getPrintableString(Printable... printables) {
+        StringBuilder fullString = new StringBuilder();
+        for (Printable p : printables) {
+            fullString.append(", ").append(p.getPrintableString());
+        }
+        return fullString.delete(0,2).toString();
```

Comment:
Perhaps you may want to look into `StringJoiner` to make the code more readable?